SettingsDialog.cs

```diff
@@ -357,6 +357,7 @@ private void InitializeComponent()
             this.btnChangeLocalDBFile.Name = "btnChangeLocalDBFile";
             this.btnChangeLocalDBFile.Size = new System.Drawing.Size(24, 20);
             this.btnChangeLocalDBFile.TabIndex = 9;
+            this.btnChangeLocalDBFile.Click += new System.EventHandler(this.btnChangeLocalDBFile_Click);

```

Comment:
This line is what actually addressed issue #4 - the rest is just some additional cleanup while I was in here investigating the settings dialog.